tests/table/test_datamining.py

```diff
@@ -30,6 +32,17 @@ def setUp(self):
     def tearDown(self):
         self.project.remove_job(self.table.name)
 
+    def test_analysis_project(self):
+        self.assertIs(self.project, self.table.analysis_project)
+        self.assertEqual(self.project.path, self.project.load(self.table.name).analysis_project.path)
+
+    def test_analysis_project_no_valid_path(self):
+        self.table.project_hdf5['input/project']['path'] = '/some/wrong/path'
+        self.table.to_hdf()
+
+        table_loaded: pyiron_base.TableJob = self.project.load(self.table.name)
+        self.assertIs(table_loaded.analysis_project, None)
+
```

Comment:
```suggestion
```
I do not have time right now to correctly get an unfitting pyiron table right now to test the warning... Thus, I take it out for now.